bin/parse-server

```diff
@@ -10,26 +10,28 @@ if (process.env.PARSE_SERVER_OPTIONS) {
 	options = JSON.parse(process.env.PARSE_SERVER_OPTIONS);
 
 } else {
-
-	options.databaseURI = process.env.PARSE_SERVER_DATABASE_URI;
-	options.cloud = process.env.PARSE_SERVER_CLOUD_CODE_MAIN;
-	options.collectionPrefix = process.env.PARSE_SERVER_COLLECTION_PREFIX;
+  options.app                        = {};
```

Comment:
Yeah I've refactored all the singletons out in a fork of mine, but it's a pretty major revision. I've harped on it in a few issues, but never gotten any traction. Basically you have to pass the context down via the Express req object. So if you need to access the MailProvider you have to do something like `req.Parse.Server.MailProvider.getAdapter()`. This is also cool because you can write custom middleware when running ParseServer has an express plugin. 

I'm not sure the singletons are blocking multiple servers though. Once we map applications to adapters then I think we're clear to run multiple servers each with independent adapter configurations for the same application. I was strongly against singletons, but I started to look at the Mongoose source and figured if Mongoose is using them, then we're probably safe.